CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ include(GetVersion)
 GetVersion(${PROJECT_SOURCE_DIR} "DPSIM")
 
 option(WITH_EIGEN_MODULE "Build with Eigen as module" OFF)
+#option(WITH_KLU "Build with KLU as module" OFF)
```

Comment:
Why is this commented out?

Reply:
The current KLU implementation does not work with WIN_32. This is a FIXME.